src/Auth0.AuthenticationApi/AuthenticationApiClient.cs

```diff
@@ -494,6 +518,31 @@ public Task<PushedAuthorizationRequestResponse> PushedAuthorizationRequestAsync(
                 cancellationToken: cancellationToken
             );
         }
+        
+        /// <inheritdoc/>
+        public Task<AssociateNewAuthenticatorResponse> AssociateNewAuthenticatorAsync(AssociateNewAuthenticatorRequest request, CancellationToken cancellationToken = default)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+            if (!request.IsValid(out List<string> validationErrors))
```

Comment:
I am not sure I understand why we are suddenly starting to do this without any context.

Reply:
removed this